tests/Doctrine/Tests/DBAL/Platforms/SQLServer2012PlatformTest.php

```diff
@@ -189,6 +189,19 @@ public function testModifyLimitQueryWithFromColumnNames(): void
         );
     }
 
+    public function testGeneratesSqlSnippets(): void
+    {
+        self::assertEquals('CONVERT(date, GETDATE())', $this->platform->getCurrentDateSQL());
+        self::assertEquals('CONVERT(time, GETDATE())', $this->platform->getCurrentTimeSQL());
+        self::assertEquals('CURRENT_TIMESTAMP', $this->platform->getCurrentTimestampSQL());
+        self::assertEquals('"', $this->platform->getIdentifierQuoteCharacter());
+
+        self::assertEquals(
+            '(column1, column2, column3)',
```

Comment:
Isn't the expected result
```suggestion
            CONCAT'(column1, column2, column3)',
```
?

Reply:
@SenseException: whoops, how embarrassing! You are correct. This has been fixed.